### ISSUE TITLE
Error loading `init.fnl` looking in `.config/fnlm/fnl` instead of `.config/nvim/fnl`

### ISSUE THREAD

**Issue by kejadlen** [rktjmp/hotpot.nvim#129]
I'm getting this on running neovim:

```
Error detected while processing /Users/alpha/.dotfiles/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /Users/alpha/.dotfiles/.config/nvim/init.lua:2: module 'init' not found:...hare/nvim/site/pack/alpha/start/hotpot/lua/hotpot/fs.lua:3: fs.read-file! i
o.open failed:/Users/alpha/.config/fnlm/fnl/init.fnl

        no file './init.lua'
        no file '/opt/homebrew/share/luajit-2.1/init.lua'
        no file '/usr/local/share/lua/5.1/init.lua'
        no file '/usr/local/share/lua/5.1/init/init.lua'
        no file '/opt/homebrew/share/lua/5.1/init.lua'
        no file '/opt/homebrew/share/lua/5.1/init/init.lua'
        no file './init.so'
        no file '/usr/local/lib/lua/5.1/init.so'
        no file '/opt/homebrew/lib/lua/5.1/init.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        /Users/alpha/.dotfiles/.config/nvim/init.lua:2: in main chunk
```

I've worked around this for now by symlinking `.config/fnlm` to `.config/nvim`, but any suggestions for where to look to figure out what's happening here would be appreciated!

**Comment by rktjmp** [comment]
Is this an up to date representation of your config, last updated 3 weeks ago? https://github.com/kejadlen/dotfiles/tree/main/.config/nvim 

Are you starting nvim with any custom NVIM_APP_NAME? Are you using any `.hotpot.lua` file?

**Comment by rktjmp** [comment]
This is an issue because your file is called `init.fnl` but is not inside a subdirectory, eg normally you would have `x/init.fnl`, then `require x` would require that file.

For now you can work around it by renaming `init.fnl` to something else (eg: `config.fnl`) then requiring that instead while I look at a fix.

**Comment by rktjmp** [comment]
Should be fixed in 9d40515

**Comment by kejadlen** [comment]
Thanks, confirmed that it works: https://git.kejadlen.dev/alpha/dotfiles/commit/abc86643ee1af5dbaeffb94e7f9ee44e5c656ca6

And appreciate the effort of finding my dotfiles, even! 😲
